packages/api/src/routes/medical/internal-cq.ts

```diff
@@ -10,35 +11,35 @@ import { Config } from "../../shared/config";
 import { asyncHandler } from "../util";
 
 const apiKey = Config.getCQApiKey();
-const apiMode = Config.getEnvType();
 
 dayjs.extend(duration);
 
 const router = Router();
 
 /**
- * GET /internal/carequality
+ * POST /internal/carequality/rebuild-directory
  *
  * Retrieves organizations from the Carequality Directory and uploads them into our database.
  * @returns Returns the number of organizations fetched, how many are newly-added and how many updated.
  */
-router.get(
-  "/",
+router.post(
+  "/rebuild-directory",
```

Comment:
nit: wondering if we're not expecting to have multiple endpoints related to the directory... in which case this would make sense, I think:
- GET /internal/carequality/directory
- POST /internal/carequality/directory/rebuild
- GET /internal/carequality/directory/:oid
- ...